invoices/invoice_expiry_watcher.go

```diff
@@ -119,20 +118,54 @@ func (ew *InvoiceExpiryWatcher) AddInvoice(
 	}
 
 	expiry := invoice.CreationDate.Add(realExpiry)
-
-	log.Debugf("Adding invoice '%v' to expiry watcher, expiration: %v",
-		paymentHash, expiry)
-
-	newInvoiceExpiry := &invoiceExpiry{
+	return &invoiceExpiry{
 		PaymentHash: paymentHash,
 		Expiry:      expiry,
 	}
+}
+
+// AddInvoices adds multiple invoices to the InvoiceExpiryWatcher.
+func (ew *InvoiceExpiryWatcher) AddInvoices(
+	invoices []channeldb.InvoiceWithPaymentHash) {
+
+	var invoicesWithExpiry []*invoiceExpiry
+	for idx := range invoices {
```

Comment:
Any particular reason you opted for indices here rather than the range loop (to pick up the item in a temp loop var). Mostly a style thing.

Reply:
Done.